packages/mdc-shape/mdc-shape.scss

```diff
@@ -20,39 +20,4 @@
 // THE SOFTWARE.
 //
 
-.mdc-shape-container {
-  display: inline-block;
-  position: relative;
-  overflow: hidden; // Prevent overflowed borders from encroaching on surrounding elements
-}
-
-.mdc-shape-container__corner {
-  position: absolute;
-  // IE and Edge require z-index set to 1 to properly mask ripple.
-  z-index: 1;
-}
-
-.mdc-shape-container__corner::before,
-.mdc-shape-container__corner::after {
-  position: absolute;
-  box-sizing: border-box;
-  width: 100%;
-  height: 100%;
-  content: "";
-}
-
-.mdc-shape-container__corner--top-left {
-  transform: rotate(-45deg);
-}
-
-.mdc-shape-container__corner--top-right {
-  transform: rotate(45deg);
-}
-
-.mdc-shape-container__corner--bottom-right {
-  transform: rotate(135deg);
-}
-
-.mdc-shape-container__corner--bottom-left {
-  transform: rotate(-135deg);
-}
+// Nothing here ¯\_(ツ)_/¯
```

Comment:
Is there intended to be anything here eventually, or should we remove this file if it's going to be nothing but mixins?

If the latter, we might also need to make some tweaks to the package check script and the material-components-web package since there will be nothing to include to emit styles for shape.

Reply:
Removed this file.